sync-sources/skills/conv-lookup/scripts/conv-find.py

```
@@ -13,53 +13,55 @@
-        for match in PROJECTS_DIR.glob(f"*/{session_id}.jsonl"):
-            return str(match), "ok"
-        return str(derived), "expired"
-
-    if legacy:
-        return legacy, "expired"
-    return None, "unknown"
+    """Resolve the transcript JSONL path by delegating to the canonical pan CLI."""
+    conv_id = info.get("id")
+    command = ["pan", "conv", "jsonl", "--json", str(conv_id)]
+    try:
+        result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)